Cython/Utility/ModuleSetupCode.c

```diff
@@ -192,7 +192,8 @@
   #ifndef CYTHON_FAST_PYCALL
     // Python 3.11 deleted localplus argument from frame object, which is used in our
     // fast_pycall code
-    #define CYTHON_FAST_PYCALL (PY_VERSION_HEX < 0x030B00A1)
+    // On Python 3.10 it causes issues when used while profiling/debugging
+    #define CYTHON_FAST_PYCALL (PY_VERSION_HEX < 0x030A0000)
```

Comment:
Shouldn't this perhaps be guarded with a debug-flag for python 3.10 to not regress performance in release builds?

Reply:
No. Unfortunately it's Python-level debugging that causes the problem so it needs applying generally

Reply:
I wonder if the difference shows up in benchmarks?